agreements.md

```diff
@@ -149,6 +149,10 @@ Fortunately, so far, there have been no such situations.
 It is not allowed to use audio or video processing in company Zoom calls.
 The only exception to this rule is the active noise cancellation function, which staff members may use freely.
 
+The reason we decided to make this policy was that it was distracting for some people.
+We also noticed that poor video processing without green screen and with loose hair causes flickering and it often hides half of someones face behind the virtual background.
+This also happens with blur.
```

Comment:
I'm good with this as written, but here is shorter version if we want to keep word count low while conveying the same information:
```suggestion
Video processing, such as virtual background, or even blur often causes flickering and face distortions.
Some of us found this very distracting.
```

Reply:
I think it has been proven that for some people this can be important both ways. Eliminating the mentioning of blur and explanation of when the issues occur might save us a line of text but might trigger a 1h discussion again. I'd rather keep the extra line.